src/hooks/useApproveCallback.ts

```diff
@@ -51,25 +55,30 @@ export function useApproveCallback(
 
   const approve = useCallback(async (): Promise<void> => {
     if (approvalState !== ApprovalState.NOT_APPROVED) {
+      toastError(t('Error'), t('Approve was called unnecessarily'))
       console.error('approve was called unnecessarily')
       return
     }
     if (!token) {
+      toastError(t('Error'), t('No token'))
       console.error('no token')
       return
     }
 
     if (!tokenContract) {
+      toastError(t('Error'), t('TokenContract is null'))
```

Comment:
this kind of message isn't user-friendly. I'm not sure if we should show it to user. Or at least we should have another meaningful message

Reply:
Maybe we should put generic error message instead?

Reply:
I think there are 2 purposes of throwing error:
 1. User knows what to do after seeing the error
 2. or User share the error message to support team, and they know what to do.
 
You could use your judgement based on above criteria to decide what best for users in this case.

Reply:
I think latter is better, what do you think ?

Reply:
That sounds great to me. Maybe `Cannot find contract of the token %tokenAddress%`

Reply:
Done